war/src/main/webapp/help/system-config/master-slave/usage.html

```diff
@@ -6,7 +6,8 @@
       Utilize this node as much as possible
```

Comment:
Ah, I was confusing this ("Usage") with "Launch method", which is already dynamic.

I will change this to "Use", as on the code side it is just a very simple `.properties` change.